modules/virt-virtctl-commands.adoc

```diff
@@ -73,6 +73,15 @@ You can use `virtctl create` commands to create manifests for virtual machines,
 |`virtctl create vm --name <vm_name>`
 |Create a VM manifest, specifying a name for the VM.
 
+|`virtctl create vm --user <user_name> --ssh-key\|password-file=<value>`
+|Create a VM manifest with a cloud-init configuration to create the selected user and either add an SSH public key from the supplied string, or a password from a file.
+
+|`virtctl create vm --access-cred type:<type>,src:<secret>,user:<user_name>`
+|Create a VM manifest with SSH credentials injected from the selected secret. Supported types are `ssh` for an SSH public key and `password` for a user password.
+
+|`virtctl create vm --volume-sysprep src:<config_map>`
+|Create a VM manifest, specifying a ConfigMap to use as the sysprep volume.
```

Comment:
You could add a note that this ConfigMap needs to contain a valid `unattend.xml`/`autounattend.xml` or also called answers file.

Reply:
Added.